pkg/controller/pod_driver.go

```diff
@@ -705,3 +716,69 @@ func (p *PodDriver) OverwirteMountPodResourcesWithPVC(ctx context.Context, pod *
 	pod.Spec.Containers[0].Resources = resources
 	return nil
 }
+
+// checkingMountPodStuck check mount pod is stuck or not
+// maybe fuse deadlock issue, they symptoms are:
+//
+// 1. pod in terminating state
+// 2. after max(pod.spec.terminationGracePeriodSeconds, 1min), the pod is still alive
+// 3. /sys/fs/fuse/connections/$dev_minor/waiting > 0
+//
+// if those conditions are true, we need to manually abort the fuse
+// connection so the pod doesn't get stuck.
+// we can do this to abort fuse connection:
+//
+//	echo 1 >> /sys/fs/fuse/connections/$dev_minor/abort
+func (p *PodDriver) checkingMountPodStuck(pod *corev1.Pod) {
```

Comment:
maybe use standard verb tense?

```suggestion
func (p *PodDriver) checkMountPodStuck(pod *corev1.Pod) {
```